welcomescreen.py

```diff
@@ -214,27 +226,33 @@ def make(self, gameDisplay, back):
                     press=0
                 '''
 
-            gameDisplay.blit(fruit, (780, 20))
+            gameDisplay.blit(fruit, ((780 + SHIFT_CORRECTOR) * scale_x,
+                                     20 * scale_y))
 
             head1 = font1.render(_("STICK"), 1, (black))
-            gameDisplay.blit(head1, (500, 20))
+            gameDisplay.blit(head1, ((500 + SHIFT_CORRECTOR) * scale_x,
+                                     20 * scale_y))
 
             head2 = font1.render(_("HERO"), 1, (black))
-            gameDisplay.blit(head2, (510, 80))
-
-            # fruitscores=font2.render(str(fruitscore),1,(0,0,0))
-            # gameDisplay.blit(fruitscores,(770+fruitscoreshift,13))
+            gameDisplay.blit(head2, ((510 + SHIFT_CORRECTOR) * scale_x,
+                                     80 * scale_y))
 
             # left and right black background patches
 
-            pygame.draw.rect(gameDisplay, black, (0, 0, 350, 768))
+            pygame.draw.rect(gameDisplay, black,
+                             (0, 0, (350 + SHIFT_CORRECTOR) * scale_x,
+                              768 * scale_y))
 
-            pygame.draw.rect(gameDisplay, black, (840, 0, 693, 768))
+            pygame.draw.rect(gameDisplay, black,
+                             ((840 + SHIFT_CORRECTOR) * scale_x, 0,
+                              (693 + SHIFT_CORRECTOR)* scale_x,
+                              768 * scale_y))
 
             pygame.display.update()
             clock.tick(60)
 
-            if crashed == True:                                   # Game crash or Close check
+            if crashed == True:
+                # Game crash or Close check
                 pygame.quit()
                 sys.exit()
```

Comment:
I would suggest `sys.exit()`  to be removed. Using `sys.exit` is not probably the right way to exit the loop. Use a return to terminate the loop gracefully. (I am not sure, if its applicable under this PR, but anyway, it would have to be removed before the last python2 release, I suppose)

Reply:
Thanks for reviewing! :)
I don't agree. welcomescreen.py , rules.py and scorescreen.py return certain values. These values are used in if constructs and have a meaning. You can see [here](https://github.com/sugarlabs/stick-hero-activity/blob/52c16c851313c1acce157fe3898c4d286356f4be/main.py#L357)  and [here](https://github.com/sugarlabs/stick-hero-activity/blob/52c16c851313c1acce157fe3898c4d286356f4be/main.py#L1036) too. Using a return statement instead of `sys.exit()` might complicate how a contributor interprets code.

Also I'm not aware of the benefits of using `return` instead of `sys.exit()`. Could you please tell me why you suggested this, might help in some other activity :)